src/components/Articles.jsx

```diff
@@ -0,0 +1,31 @@
+import react from "react";
+import React from "react";
+import { useState, useEffect } from "react";
+import { getArticles } from "../utils/axios";
+import ArticleCard from "./ArticleCard";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(false);
+  useEffect(() => {
+    setLoading(true);
```

Comment:
As you have no dependencies in your array on line 17, then you don't need this setLoading(true), you could just initalise the state on line 9 to start as true, instead of false